config.ml

```diff
@@ -53,6 +53,14 @@ let disk =
 
 (* Command-line options *)
 
+let root_k =
+  let doc = Key.Arg.info ~doc:"Blog URL" ["root"] in
+  Key.(create "root" Arg.(opt string "http://localhost" doc))
```

Comment:
or maybe something where the default is built dynamically based on the values of port and tls?